app/controllers/api/v1/tokens_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class TokensController < Doorkeeper::TokensController
+      include ErrorHandlerConcern
+
+      after_action :handle_jsonapi_success
+
+      private
+
+      # Overwrite this method as it bypass the custom_error_response provided by Doorkeeper
+      def revocation_error_response
+        error_description = I18n.t(:unauthorized, scope: %i[doorkeeper errors messages revoke])
+        {
+          errors: build_error(detail: error_description, code: :invalid_client)
+        }
+      end
+
+      def handle_jsonapi_success
```

Comment:
@malparty Firstly, if you need a callback only specific to a action, you can 
```
   after_action : handle_jsonapi_success, only: :revoke
```

Secondly, IMO going this way  of setting response in after_action is a bit confusing and we usually dont do this in rails proejcts. If they dont provide a single method we could override, we can override the whole action `revoke`.

Reply:
Yes, I had a first version with the `only: : revoke` keyword. But Rubocop does not allow it (as the revoke action is not defined in the same file = Rubocop indicates that this is confusing for developers). Which I understand. 

Point 2: Yes, I agree it's an ugly hack!! And so I'll go for the re-write of the whole action :) Thanks!

Reply:
Ok, done @ankitkalia1195  ;-)